src/pysdmx/model/message.py

```diff
@@ -4,8 +4,13 @@
 can be written.
 """
 
+from datetime import datetime
 from enum import Enum
 
+from msgspec import Struct
+
+from pysdmx.errors import ClientError
+
 
 class MessageType(Enum):
```

Comment:
The name might be a bit confusing as, with MessageType, some might understand Structure, Data, ReferenceMetadata, etc. Or even something more specific like StructureSpecificData2.1, GenericData2.0, etc., etc. Maybe we could rename it to SdmxAction?

Reply:
@sosna Need another name different than SdmxAction, it has to be some name that refers to the type of message we want to write (data, structures, etc)

Reply:
The Message class should, theoretically, be able to write also Reference Metadata or any type of SDMX classes, including Submission Results and Errors (from FMR and SDMX APIs). I would prefer to keep MessageType or try to find a synonym that explains better this capability.

Reply:
Thanks, Javier. It's all clear now and, based on what you described, I agree the name is a good one 😊.

However, as it seems to be strongly related to writing messages (which I didn't know when making my comment), maybe it could be moved below the io module, or even io.sdmx21, as it seems to be SDMX-ML related?

Reply:
Yes, makes sense to move it to io.xml at least for now, I cannot think of any other place where this could be used, AFAIK the other flavours of SDMX does not have any of these values (support for Submission and Error messages, differentiation between Generic and Structure Specific...). The value of this Enum must be specified when writing only to SDMX ML files.

Reply:
Moved MessageType to io.xml.enums.py